gitea/release.go

```diff
@@ -28,6 +28,7 @@ type Release struct {
 	// swagger:strfmt date-time
 	PublishedAt  time.Time `json:"published_at"`
 	Publisher    *User     `json:"author"`
+	Attachments  []*Attachment `json:"assets"`
```

Comment:
I removed this because the field was never populated in API endpoints. Releases that do in fact have attachments had an empty `assets` fields, which I thought was misleading.

I think we should only add it back if we have plans to populate it in the near future.

Reply:
Looks like we do have plans: https://github.com/go-gitea/gitea/pull/3075